src/test/java/org/sagebionetworks/bridge/services/Exporter3ServiceTest.java

```diff
@@ -1550,6 +1569,101 @@ public void completeUpload_NoAccount() throws Exception {
         verifyZeroInteractions(mockHealthDataEx3Service, mockSqsClient);
     }
 
+    @Test
+    public void exportTimelineForStudy_1stTime() throws Exception {
```

Comment:
We should also verify the file content and the markdown page that we uploaded to Synapse.